Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerDevolvedRegulation.cshtml

```diff
@@ -0,0 +1,64 @@
+@page "/organisation/{id}/supplier-to-buyer-devolved-regulation"
+@using CO.CDP.OrganisationApp.Constants
+
+@model CO.CDP.OrganisationApp.Pages.Organisation.SupplierToBuyerDevolvedRegulationModel
+@{
+    ViewData["Title"] = "Do devolved regulations apply to your organisation?";
+    var devolvedHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Devolved)).HasInnerHtml;
+
+    var backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-devolved-regulation";
```

Comment:
Is this backlink going to the same place as we are already at?

Reply:
well spotted, 
It should be going back to organisation type screen
UPDATED: var backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-organisation-type";